lib/bot.js

```diff
@@ -135,16 +136,20 @@ class Bot {
     logger.debug('Channel Mapping', channelName, this.channelMapping[channelName]);
     if (ircChannel) {
       const username = author.username;
-      const colorIndex = (username.charCodeAt(0) + username.length) % NICK_COLORS.length;
-      const coloredUsername = irc.colors.wrap(NICK_COLORS[colorIndex], username);
       let text = this.parseText(message);
+      let newUsername = username;
+      if (this.ircNickColor) {
+        const colorIndex = (username.charCodeAt(0) + username.length) % NICK_COLORS.length;
+        const coloredUsername = irc.colors.wrap(NICK_COLORS[colorIndex], username);
```

Comment:
Don't need coloredUsername here, can just do `newUsername = irc.colors.wrap...`. I think something like `displayUsername` would be a better variable name than `newUsername` though :)